dev-support/ci/selective_ci_checks.bats

```diff
@@ -66,6 +66,17 @@ load bats-assert/load.bash
   assert_output -p needs-kubernetes-tests=true
 }
 
+@test "runner image update" {
+  run dev-support/ci/selective_ci_checks.sh b95eeba82a
+
+  assert_output -p 'basic-checks=["rat"]'
```

Comment:
Is it correct to say that without these changes the basic-checks array would have looked like this?:
```
     basic-checks=["rat","checkstyle","findbugs","unit"]
```

Reply:
Correct.  The removed ones are all java-specific.